core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/edit-ema-palette/components/edit-ema-palette-contentlets/edit-ema-palette-contentlets.component.html

```diff
@@ -32,7 +32,7 @@
                     *ngFor="let contentlet of contentlets.items; let i = index"
                     [attr.data-item]="
                         {
-                            identifier: contentlet.identifier,
+                            contentlet: { identifier: contentlet.identifier },
```

Comment:
```suggestion
                            contentlet: { contentlet },
```

Why not? too much content?

Reply:
Not sure why I did that tho

Reply:
I reverted this to the original, for some reason it broke everything. Not sure why